changelog/fragments/add-componentconfig-flag.yaml

```
@@ -0,0 +1,26 @@
+entries:
+  - description: >
+      (ansible/v1, helm/v1) correctly support configuring the manager by file using the
+      `--config` flag. This flag was not added to either ansible-/helm-operator binary
+      when file support was originally added.
```

```suggestion
      (ansible/v1, helm/v1) fix the issue to support component config added for in the release 1.6.1 by adding the missing flag `--config` to its binaries and images.
```

could we not simplify?